taglib/riff/wav/wavfile.cpp

```diff
@@ -47,7 +48,8 @@ class RIFF::WAV::File::FilePrivate
     properties(0),
     tagChunkID("ID3 "),
     hasID3v2(false),
-    hasInfo(false)
+    hasInfo(false),
+    acidInfo(NULL)
```

Comment:
That I did not know was an issue ... Thank you.
I did test on MinGW, MSVC2010, FreeBSD-8.2 (GCC 4.2.1).

I will follow your suggestion and change it to Zero.